src/main/kotlin/OOP/Lior/OOPLiorLec06.kt

```diff
@@ -0,0 +1,78 @@
+package OOP.Lior
+
+//- Add a class Participant.kt - each participant has a name (Name), email. Create getter and setter
+data class Participant(val name: String, val email: String)
+
+//- Add class Meeting.kt - each meeting need to know name and location
+open class Meeting(val name: String, val location: Location)
+{
+    private val _participants = mutableListOf<Participant>()
```

Comment:
what is the purpose of adding it as a private val?

Reply:
I try to avoid using mutable lists and prefer keeping them private for better encapsulation. However, I can adjust this if needed.